ui/nuxeo-slots.js

```diff
@@ -148,7 +158,15 @@ window.nuxeo.slots.setSharedModel = (model) => {
 
     disconnectedCallback() {
       super.disconnectedCallback();
-      this._unregister(this.slot);
+      this._unregister(this.name || this.slot);
+    }
+
+    _slotChanged(newSlot, oldSlot) {
+      if (this.name) {
+        this._updateContent();
+      } else {
+        this._register(newSlot, oldSlot);
+      }
     }
 
     _register(newSlot, oldSlot) {
```

Comment:
perhaps we should review the name of this arguments since we're not using `name` instead of `slot`

Reply:
What about `newName` and `oldName`?